ami-build/scripts/run-fun.sh

```diff
@@ -187,8 +187,8 @@ _agent_start_finalize () {
 
 
 agent_start_build () {
-    local bazel_cache_bucket="$6" \
-          cache_prefix="$7" \
+    local bazel_cache_bucket="$5" \
+          cache_prefix="$6" \
```

Comment:
(optional, pre-existing) This is a large number of arguments. Consider documenting them and/or giving them local names.

Reply:
yeah, this is a bit non-intuitive - it passes through the args and adds a couple

i believe that the correction i made here is necessary but i need to confirm that

Reply:
on further investigation these are the same names that are used in terraform - not sure how they could be named locally other than what is there now

Reply:
Ack on running being the only known way to test this.

On the documentation - can we add a function doc for `agent_start_build` that would list the arguments this function takes and their meaning?

Reply:
added